users/models.py

```diff
@@ -32,12 +33,32 @@ class Meta:
         verbose_name_plural = "Allergies"
 
 
+class KnightHood(models.Model):
+    """
+    Model for knighthood. A knighthood is a special title given to a user
+    """
+
+    user = models.OneToOneField(
+        "users.User", on_delete=models.CASCADE, related_name="knighthood"
+    )
+    knighted_at = models.DateField(default=date.today())
```

Comment:
Litt usikker om dette funker med default, du må sende inn sfunksjonen og ikke "invoke" den om det gir mening, ref https://stackoverflow.com/questions/2029295/django-datefield-default-options.

Ser også at det er `auto_now_add=True` i migrasjonen så stemmer denne modellen overens med det?

Reply:
Her har jeg bare somlet først med å bruke auto_now, og så fjernet jeg det igjen for å replace med default

Reply:
Så hyggelig av django å gi meg en warning på det, tross at det står i docs:
![image](https://github.com/user-attachments/assets/447bda8d-f13c-4ada-8ec9-7104a12d679a)

Reply:
Dem klager på at du bruker `today()` og ikke `today` som samsvarer med docsene hehe